Azure/sonic-buildimage
# [openconfig_acl] Add SONiC ACL extension to open config ACL model

- Add support for VLAN ID match
- Add support for ICMP type/code match

Signed-off-by: Danny Allen <daall@microsoft.com>



#### Why I did it
To allow users to add ACL rules w/ ICMP and VLAN qualifiers via acl-loader.

#### How I did it
Added `sonic-acl-extension.yang` to augment the existing OpenConfig spec and used pyangbind to generate a new set of python bindings.

#### How to verify it
Re-run the ACL tests w/ the new openconfig_acl.py script and verify that all rules are still loaded properly and dataplane behavior is not impacted.

#### Which release branch to backport (provide reason below if selected)



- [] 201911

#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
